### ISSUE TITLE
FYI, `SwiftUI-Introspect` 0.2.0 changed min iOS to 13, breaking builds

### ISSUE THREAD

**Issue by ggrell** [Amzd/ScrollViewProxy#20]
FYI, wanted to let you know that projects including `ScrollViewProxy` will likely break now that https://github.com/siteline/SwiftUI-Introspect/releases/tag/0.2.0 has a minimum iOS 13, but this project has min iOS 11. Can either try and upgrade this project to use the latest introspect library, or update the Package.swift to only include the exact version "0.1.4".

**Workaround**:
If someone includes this package as a dependency, you must also include the correct `SwiftUI-Introspect` version before the iOS minimum changed, like this:
```swift
    dependencies: [
        ...
        .package(url: "https://github.com/Amzd/ScrollViewProxy", exact: "1.0.3"),
        .package(url: "https://github.com/siteline/SwiftUI-Introspect.git", exact: "0.1.4"),
        ...
    ]
```

**Comment by Amzd** [comment]
Hmm, that's not great. Maybe I should drop Introspect and just grab the scrollview similar to Introspect but without the rest of the package, because if I pin Introspect there will be others that want to use the newer release of it I'd assume?

**Comment by ggrell** [comment]
True, it's possible some will need newer versions. Since Introspect hasn't released v1.0, might be best to copy code until they do, unfortunately.

I reached out to the maintainers here: https://github.com/siteline/SwiftUI-Introspect/issues/206
